pyproject.toml

```diff
@@ -88,6 +88,9 @@ minversion = "6.0"
 log_cli_level = "INFO"
 markers = ["unstable"]
 asyncio_mode = "auto"
+filterwarnings = [
```

Comment:
noob q - what does this do?

Reply:
This will transform any runtime warnings of the tests in errors.
`RuntimeWarning` are warnings about dubious runtime errors such as "coroutine was never awaited".
This will help cleaning the tests and avoid some errors we have were we forget an `await`